cost/terminate_policy/instance_terminate.pt

```diff
@@ -168,18 +164,28 @@ end
 policy "pol_instances" do
   validate_each $ds_instances_with_timestamp do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Terminable Instances"
-    detail_template <<-EOS
-# Terminable Instances
-| Name | Terminate Date | Current Time | Locked |
-| ---- | -------------- | ------------ | ------ |
-{{ range data -}}
-| {{ .name }} | {{.terminate_date}} | {{.run_time}} | {{.locked}} |
-{{ end -}}
-EOS
     escalate $email
     escalate $report_and_terminate_instances
     check logic_not(val(item,"terminate"))
     resolve $report_resolution
+    export do
+      resource_level true
+      field "id" do
```

Comment:
id should be the last column.  
Add cloud first column, sorted
Add datacenter as second column sorted

Reply:
added datacenter and cloud.